docs/development/testguide.rst

```diff
@@ -67,50 +69,66 @@ options require the `pytest-pep8 plugin
 <https://pypi.org/project/pytest-pep8>`_, which must be installed
 separately.
 
-setup.py test
-=============
+.. _running-pytest:
+
+pytest
+======
+
+The test suite can be run directly from the native ``pytest`` command. In this
+case, it is important for developers to be aware that they must manually
+rebuild any extensions by running::
+
+    python setup.py build_ext --inplace
+
+before running the test with pytest with::
+
+    pytest
+
+Instead of calling ``build_ext --inplace``, you can also install the package
+in editable mode::
+
+    pip install -e .[test]
```

Comment:
Isn't this the preferred way? If so, this should be mentioned first before the `build_ext` option.

Reply:
I've switched this around